.env

```diff
@@ -61,6 +61,7 @@ MF_THINGS_DB_USER=mainflux
 MF_THINGS_DB_PASS=mainflux
 MF_THINGS_DB=things
 MF_THINGS_SECRET=secret
+MF_THINGS_ES_ACTIVE=false
```

Comment:
Do we need an extra ENV var?

Can't we say: if you did not provide value for ES server URL and/or if it is an empty string - just consider that ES is not active.

Reply:
After chat with @anovakovic01 it was concluded that empty strings can be confusing and all other defaults changes in that case (for consistency).

So we stick with this approach.